### PR TITLE
chore: Add blog link to MEOW

### PR DESCRIPTION
**Pull request by chefjackson**
---

### Detailed summary:
- Added `data-dd-action-name` attribute to several buttons in the Swap feature for tracking purposes.
- Added a link to a token campaign for the MEOW token.
- Added a "Select currency" data-dd-action-name attribute to the currency select button in the CurrencyInputPanel component.
- Added `data-dd-action-name` attributes to the "Copy token address" and "Add to wallet" buttons in the CurrencyInputPanel component.
- Added `data-dd-action-name` attributes to the balance percentage buttons in the CurrencyInputPanel component.
- Added `data-dd-action-name` attributes to the "Buy crypto", "Price chart", "Hot token list", "Swap settings", "Swap history", and "Swap refresh" buttons in the CurrencyInputHeader component.
- Added `data-dd-action-name` attributes to the "Risk scan retry button", "Swap slippage button", "Swap remove recipient button", "Swap flip button", "Swap wrap button", "Swap mm commit button", and "Swap reset routing settings button" buttons in various components.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/components/CurrencyInputPanel/index.tsx [hunk 1]
```diff
@@ -216,6 +228,19 @@ const CurrencyInputPanel = memo(function CurrencyInputPanel({
                 />
               </Flex>
             ) : null}
+            {token &&
+            tokenAddress &&
+            token.chainId === ChainId.ZKSYNC &&
+            tokenAddress === '0x79db8c67d0c33203da4Efb58F7D325E1e0d4d692' ? (
```
Comment (ChefJerry) [inline apps/web/src/components/CurrencyInputPanel/index.tsx:234]: use `zksyncTokens.meow.address` will be better

File: apps/web/src/components/CurrencyInputPanel/index.tsx [hunk 2]
```diff
@@ -216,6 +228,19 @@ const CurrencyInputPanel = memo(function CurrencyInputPanel({
...
+              <LinkExternal
```
Comment (chef-jojo) [inline apps/web/src/components/CurrencyInputPanel/index.tsx:235]: Can we add tracking for this link click?

File: apps/web/src/components/CurrencyInputPanel/index.tsx (hunk 1 above)
Comment (chef-jojo) [reply apps/web/src/components/CurrencyInputPanel/index.tsx:234]: case sensitive compare?

Comment (chefjackson) [reply apps/web/src/components/CurrencyInputPanel/index.tsx]: token address already checksumed. Should be good. Changed to use token instance to compare

**Review by chef-jojo — DISMISSED** [review]

**Pull request merged**